From d0b2e2429b71a6b69b994d602c472af991a7b039 Mon Sep 17 00:00:00 2001
From: David Sniatkiwicz <davidsn@marvell.com>
Date: Sun, 31 Dec 2017 13:41:37 +0200
Subject: [PATCH 2145/2241] mvebu: dts: cp110: fix typo in CP110_NUM on CP1

Currently, CP110_NUM on CP1 is set to '0'.
This patch fixes this issue by fixing that typo
and setting CP110_NUM on CP1 to '1'.

Change-Id: I21a966ced187a8fd6902e4ece1a65770ef7857b0
Signed-off-by: David Sniatkiwicz <davidsn@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/48281
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-8040.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040.dtsi b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
index f209afa..6e3f2b4 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
@@ -90,7 +90,7 @@
 
 /* CP110-1 Settings */
 #define CP110_NAME		cp1
-#define CP110_NUM		0
+#define CP110_NUM		1
 #define CP110_BASE              0xf4000000
 #define CP110_PCIE_MEM_BASE     0xfa000000
 #define CP110_PCIE_MEM_OFFSET	0x1000000
-- 
2.7.4

